pkg/setting/setting_unified_alerting.go

```diff
@@ -352,6 +353,11 @@ func (cfg *Cfg) ReadUnifiedAlertingSettings(iniFile *ini.File) error {
 		TenantID: remoteAlertmanager.Key("tenant").MustString(""),
 		Password: remoteAlertmanager.Key("password").MustString(""),
 	}
+	uaCfgRemoteAM.SyncInterval, err = gtime.ParseDuration(valueAsString(remoteAlertmanager, "sync_interval", (schedulerDefaultAdminConfigPollInterval).String()))
```

Comment:
This should be done directly on the configuration struct.

Reply:
All intervals taken from the .ini file are parsed [here](https://github.com/grafana/grafana/blob/fdaf6e3f2ecfe379350884f9da4e2841980da037/pkg/setting/setting_unified_alerting.go#L232-L251)

Reply:
Unless we have a really good reason I don't think we should take a different approach.

Reply:
Oh sorry, I missed this was done on the settings package 🤦 - you're good, carry on.